_bibliography/published.bib

```diff
@@ -1,3 +1,26 @@
+@Article {Hinsen:2020,
+  author =       {Konrad Hinsen},
+  title =        {{[¬Rp] Stokes drag on conglomerates of spheres}},
+  journal =      {ReScience C},
+  year =         {2020},
+  month =        {6},
+  volume =       {6},
+  number =       {1},
+  pages =        {{#7}},
+  doi =          {10.5281/zenodo.3889694},
+  url =          {https://zenodo.org/record/3889694/files/article.pdf},
+  code_url =     {https://github.com/khinsen/rescience-ten-year-challenge-paper-4},
+  code_doi =     {},
+  code_swh =     {swh:1:dir:195b7890e84d3bcdc1fdbbbc351c63ac69a64ba0},
+  data_url =     {},
+  data_doi =     {},
+  review_url =   {https://github.com/ReScience/submissions/issues/20},
+  type =         {Reproduction},
+  language =     {Fortran77},
+  domain =       {Fluid dynamics},
+  keywords =     {protein structure, protein crystals, normal modes, Python}
+}
+
 @Article {Courtès:2020,
```

Comment:
Maybe I am old-fashioned, but I don't even want to explore if Unicode is OK in bibliography labels, because I am sure there is some exotic situation where it isn't.